ocs_ci/utility/utils.py

```diff
@@ -954,6 +954,34 @@ def ocsci_log_path():
     )
 
 
+def get_testrun_name():
+    """
+    Prepare testrun ID for Polarion (and other reports).
+
+    Return config.REPORTING["polarion"]["testrun_id"], if configured.
+    Otherwise prepare testrun ID based on Upstream/Downstream information,
+    OCS version and used markers.
+
+    Returns:
+        str: String containing testrun ID
+    """
+    markers = config.RUN['cli_params'].get('-m', '').replace(" ", "-")
+    us_ds = config.REPORTING.get("us_ds")
+    if config.REPORTING["polarion"].get("testrun_id"):
```

Comment:
is testrun_id is same as run_id ?If not where do we updating testrun_id to REPORTING section

Reply:
I've renamed the `testrun_id` to `testrun_name` in b123b44, to be more consistent with the `get_testrun_name`.

`testrun_name` (previously `testrun_id`) is optional, if not provided, the *Test Run Name* is generated from other sources (see the following lines).